packages/manager/modules/pci/src/projects/project/serving/namespace/models/add/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,26 @@
+{
+  "pci_projects_project_serving_namespace_models_add_error": "Une erreur est survenue durant la création du modèle : {{ message }}",
+  "pci_projects_project_serving_namespace_models_add_success": "Votre modèle est en cours de création",
+  "pci_projects_project_serving_namespace_models_add": "Déployer un modèle",
+  "pci_projects_project_serving_namespace_models_add_type": "Type de modèle",
+  "pci_projects_project_serving_namespace_models_add_type_preset-image": "Modèle prédéfini",
+  "pci_projects_project_serving_namespace_models_add_type_preset-image_popover": "Sélectionnez parmi la liste des modèles prédéfinis",
+  "pci_projects_project_serving_namespace_models_add_type_build-image": "Modèle personnalisé",
+  "pci_projects_project_serving_namespace_models_add_type_build-image_popover": "Déployer votre modèle personnalisé",
+  "pci_projects_project_serving_namespace_models_add_name": "Nom",
+  "pci_projects_project_serving_namespace_models_add_name_help": "Le nom du modèle doit commencer par un caractère alphanumérique minuscule ou majuscule (de a à z et de 0 à 9) et peut comporter le signe de ponctuation suivant : « - »",
+  "pci_projects_project_serving_namespace_models_add_model": "Modèle",
+  "pci_projects_project_serving_namespace_models_add_instances": "Configuration des instances",
+  "pci_projects_project_serving_namespace_models_add_instances_info": "Pendant la phase de découverte un seul type d'instance est disponible. Des types d'instance additionnels seront créés pour s'adapter aux besoins spécifiques.",
+  "pci_projects_project_serving_namespace_models_add_autoscaling": "Auto Scaling",
+  "pci_projects_project_serving_namespace_models_add_instances_price": "{{price}}€ ex. HT/heure",
```

Comment:
Not everyone who reads in French uses `Euro` 🙂 

This kind of data must be retrieved from the API